Gems/EMotionFX/Code/EMotionFX/Rendering/Common/RenderUtil.cpp

```diff
@@ -10,6 +10,7 @@
 #include "RenderUtil.h"
 #include <MCore/Source/Algorithms.h>
 #include <MCore/Source/Compare.h>
+#include <MCore/Source/FastMath.h>
```

Comment:
We are moving from mcore math to using az math. For example, MCore::Math::abs should uses AZ::abs
You can find the api under code/framework/azcore/azcore/math/..